README.Rmd

```diff
@@ -16,7 +16,7 @@ Topological Data Analysis (TDA) is a recent and fast growing field providing a s
 
 ## Install Python Gudhi Library  
 
-See the [installation page](http://gudhi.gforge.inria.fr/python/latest/installation.html) or if you have conda you can make a [conda install](https://anaconda.org/conda-forge/gudhi).
+See the [installation page](http://gudhi.inria.fr/python/latest/installation.html) or if you have conda you can make a [conda install](https://anaconda.org/conda-forge/gudhi).
```

Comment:
I would prefer https links but let it like that as it also works.

Reply:
Feel free to change it to https, I just don't want to keep broken links too long...